src/App.jsx

```diff
@@ -1,12 +1,18 @@
 import { Routes, Route } from 'react-router-dom';
 import './App.css';
-import { useSelector } from 'react-redux';
+import { useSelector, useDispatch } from 'react-redux';
+import { useEffect } from 'react';
 import Library from './components/Library';
 import Categories from './components/Categories';
 import Header from './components/Header';
+import { recieveBooks } from './redux/books/books';
 
 function App() {
   const state = useSelector((state) => state);
+  const dispatch = useDispatch();
+  useEffect(() => async () => {
+    await dispatch(recieveBooks());
+  }, []);
```

Comment:
- [ ] As requested by the previous reviewer, Please remove `async` and `await` from this hook, You should not use these with `redux-thunk`. Simply remove `async` and `await` from the `useEffect` hook.